sound/hybrid_font.h

```diff
@@ -120,7 +128,37 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  
+  void StartSwing() override {
+    if (!guess_monophonic_) {
+      if (!swing_player_) {
+        swing_player_ = PlayPolyphonic(&swng);
+      }
+      // avoid overlapping swings unless at least a quarter of the swing has played.  Stops double swing events.
+      if ((swing_player_->length() - swing_player_->pos()) / swing_player_->length() <= 0.75) {
```

Comment:
I do like this, although I think we should make the constant configurable.
Also, it seems like pos / length >= 0.25 would be simpler and more clear.

SB_motion() has a different ways to avoid double swings, which is that swing starts once speed goes over a constant, but you can't trigger another swing until speed drops below that constant, and then goes above it again.

Although, we probably need a little hysteresis, so that speed has to drop below 200 or something before swinging_ is set to false again.

Reply:
That’s a good point about the threshold. I’ll play with that. 

I agree about the overlap constant. I’ll make it a config variable.

Reply:
ProffieOSSwingOverlay defined.  What should the default be?

Reply:
I've added a check in the SB_Motion function that only sets swinging to false after speed falls below 80% of the threshold.  Still have to test and see if that value works well.  Should that threshold also be a config variable?

Reply:
added new config variable ProffieOSSwingLowerThreshold, set to .8 by default.  This works great on monophonic fonts to prevent double swings and still be responsive.